### ISSUE TITLE
Don't print info when running webpack --json

### ISSUE THREAD

**Issue by geirsagberg** [TypeStrong/fork-ts-checker-webpack-plugin#101]
Version: 0.3.0

Running `webpack --profile --json` currently produces the following when `fork-ts-checker-webpack-plugin` is included:

```
Starting type checking service...
Using [1m1 worker[22m with [1m2048MB[22m memory limit
{
  "errors": [],
  "warnings": [],
  "version": "3.10.0",
  "hash": "8c0541347f7cefe0876b",
...
```

The first two lines cause e.g. `webpack-bundle-analyzer` to fail.

When webpack is ran with `--json`, nothing should be written to console.

**Comment by johnnyreilly** [comment]
We had this same issue in ts-loader for ages. In the  end we bowed to peer pressure and dialed down our logging to not emit by default. Before that we had a `--silent` option which nixed it - people could use that and did. Maybe worth introducing a similar option?

**Comment by meyer** [comment]
yep, same issue here. Are you all accepting PRs?

**Comment by meyer** [comment]
I just found [the `silent` option](https://github.com/Realytics/fork-ts-checker-webpack-plugin/blob/477a66fd038236060219512c8cbb60968d94f8e5/src/index.ts#L118). Looks like it does what I want.

**Comment by Akuukis** [comment]
Got the same problem, this works for me:
```js
new ForkTsCheckerWebpackPlugin({tsconfig: 'tsconfig.json', silent: process.argv.includes('--json')}),
```

**Comment by piotr-oles** [comment]
@geirsagberg , @meyer , @Akuukis
Please try [fork-ts-checker-webpack-plugin@alpha](https://github.com/TypeStrong/fork-ts-checker-webpack-plugin/tree/alpha) - I've published a new version which should resolve this issue 🚀
I will close this issue to clean-up the backlog :)
